plugins/modules/helm_repository.py

```diff
@@ -72,6 +72,8 @@
     default: false
     type: bool
     version_added: 2.3.0
+extends_documentation_fragment:
```

Comment:
This is a bit confusing because the related doc mentions the parameters ``api_key``, ``validate_certs``, ``ca_cert`` added in version ``1.2.0``, but with this modification should be something like ``2.2.4``